streaming/java/streaming-state/src/main/java/org/ray/streaming/state/keystate/state/impl/ValueStateImpl.java

```diff
@@ -0,0 +1,30 @@
+package org.ray.streaming.state.keystate.state.impl;
+
+import org.ray.streaming.state.backend.KeyStateBackend;
+import org.ray.streaming.state.keystate.desc.ValueStateDescriptor;
+import org.ray.streaming.state.keystate.state.ValueState;
+
+/**
+ * ValueState implementation.
+ */
+public class ValueStateImpl<T> extends AbstractState<T> implements ValueState<T> {
+
+  public ValueStateImpl(ValueStateDescriptor<T> descriptor, KeyStateBackend backend) {
+    super(backend, descriptor);
+  }
+
+  @Override
+  public void update(T value) {
+    this.put(descriptor, value);
+  }
+
+  @Override
+  public T get() {
+    T value = this.get(descriptor);
+    if (value == null) {
```

Comment:
use `null == value` to avoid assign null to variable.
Same issue in other places.

Reply:
Ok，I'll fix this